phpunit.xml.dist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         syntaxCheck="false"
+         bootstrap="vendor/autoload.php"
+>
+    <testsuites>
+        <testsuite name="ValueObject testsuite">
+            <directory>./tests/</directory>
```

Comment:
actually the dir name We created is 'test' without "s" in the end.

change dir name and namespace in composer.json ...or change directory name in phpunit.xml.dist

I think first option is better